sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -315,3 +354,16 @@ class UrReturnHelper {
   void *param_value;
   size_t *param_value_size_ret;
 };
+
+// Global variables for ZER_EXT_RESULT_ADAPTER_SPECIFIC_ERROR
+constexpr size_t MaxMessageSize = 256;
+extern thread_local zer_result_t ErrorMessageCode;
+extern thread_local char ErrorMessage[MaxMessageSize];
+
+// Utility function for setting a message and warning
+[[maybe_unused]] void setErrorMessage(const char *message,
+                                      zer_result_t error_code);
+
+// Returns plugin specific error and warning messages
+// TODO: promote to Unified Runtime API

```

Comment:
do we have an issue already opened for promoting this interface to the UR? I have some comments on it.

Reply:
@jandres742 : I did not create one assuming it already exists since that was a known gap
@pbalcer: can you point to the issue or create a new one?

Reply:
If you're referring to GetLastError then I belive this is already implemented: https://github.com/oneapi-src/unified-runtime/issues/16

Reply:
Thanks @igchor for the link.

Reply:
Sounds like it. So this would be solved by updating to the latest UR